config/production.json

```diff
@@ -17,6 +17,7 @@
 		"code-splitting": true,
 		"community-translator": true,
 		"desktop-promo": true,
+		"google-voucher": true,
```

Comment:
It was enabled some days ago @mtias 
https://github.com/Automattic/wp-calypso/blob/master/config/development.json#L41